app/js/books_page/books_page_ctrl.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
-class BooksPageCtrl {
-  constructor() {
-    this.searchText = '';
-  }
+function BooksPageCtrl() {
+  let vm = this;
```

Comment:
ViewModel
It's more of a view model than a conventional controller.

But now that I think about it, I'm already using "ctrl" in the views. So i should make it more consistent.
Which do you prefer? We're more accustomed to controller, but Angular's "controller" acts as a view model